zuul-core/src/test/java/com/netflix/zuul/context/SessionContextTest.java

```diff
@@ -107,4 +105,18 @@ void remove() {
         Truth.assertThat(context.get(key)).isNull();
         Truth.assertThat(val).isEqualTo("bar");
     }
+
+    @Test
+    void containsKey() {
```

Comment:
maybe add a test case for the non-typed version as well (if it doesn't already exist)?